pages/spicedb/concepts/datastores/cloud-spanner.mdx

```diff
@@ -0,0 +1,44 @@
+import { Callout } from 'nextra/components'
+
+# Cloud Spanner
+
+## Usage Notes
+
+- Requires a Google Cloud Account with an active Cloud Spanner instance
+- Take advantage of Google's TrueTime.
+  The Spanner driver assumes the database is linearizable and skips the transaction overlap strategy required by CockroachDB.
```

Comment:
> The Spanner driver assumes the database is linearizable

is this something the user can change?

> skips the transaction overlap strategy required by CockroachDB.

what does this mean? like if i'm a spicedb newbie user do i need to configure anything to prevent the New Enemy problem or am i protected by default?